src/management-system-v2/components/processes.tsx

```diff
@@ -76,9 +91,33 @@ const Processes: FC = () => {
           }}
         />
       </Tooltip>
-      <Tooltip placement="top" title={'Delete'}>
-        <DeleteOutlined />
-      </Tooltip>
+      {selectedRowKeys.length === 1 && (
+        <ProcessEditButton
+          definitionId={selectedRowKeys[0] as string}
+          wrapperElement={
+            <Tooltip placement="top" title={'Edit'}>
+              <EditOutlined onClick={() => {}} />
+            </Tooltip>
+          }
+          onEdited={() => {
+            refetchProcesses();
+          }}
+        />
+      )}
+      <AuthCan resource="Process" action="delete">
```

Comment:
since it's a specific process you should use `resource={toCaslResource('Process', object)}`